src/index.js

```diff
@@ -1,131 +1,11 @@
-function _foldToObject(memo, [k, v]) {
-  memo[k] = v;
-  return memo;
-}
+import implementation from './implementation';
+import getPolyfill from './polyfill';
+import shim from './shim';
 
-function _objectMap(obj, fn) {
-  return _entries(obj).map(fn).reduce(_foldToObject, {});
-}
+Object.defineProperties(implementation, {
+  implementation: { value: implementation, enumerable: false, configurable: true, writable: false },
```

Comment:
https://npmjs.com/define-properties will do this for you, with a fallback for when Object.defineProperties isn't supported